src/index.html

```diff
@@ -5,8 +5,413 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+  
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img 
+          src="images/logo.png"
+          alt="Moyo Logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__link is-active" href="#apple">
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops&computers"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="card">
+      <div class="card__form" data-qa="card">
```

Comment:
```suggestion
    <main class="catalog">
      <div class="card catalog__card" data-qa="card">
```
it should work like this

Reply:
done